modules/apps/data-engine/data-engine-taglib/jest-setup.config.js

```diff
@@ -43,6 +43,7 @@ window.util = {
 	...window.util,
 	escape: (data) => data,
 	selectEntity: () => {},
+	sub: (data) => data,
```

Comment:
instead of adding the `sub` here, maybe it should be:

```suggestion
	...window.Liferay.Util,
	escape: (data) => data,
	selectEntity: () => {},
```

But since you added the other mocks to `npm-scripts` removing everything will be better.

Reply:
Yes, the [PR](https://github.com/liferay/liferay-frontend-projects/pull/619) is open